stdlib/Random/src/RNGs.jl

```diff
@@ -181,9 +181,9 @@ function show(io::IO, rng::MersenneTwister)
     seed = from_seed(rng.seed)
     seed_str = seed <= typemax(Int) ? string(seed) : "0x" * string(seed, base=16) # DWIM
     if rng.adv_jump == 0 && rng.adv == 0
-        return print(io, "MersenneTwister($seed_str)")
+        return print(io, "$MersenneTwister($seed_str)")
```

Comment:
```suggestion
        return print(io, MersenneTwister, "($seed_str)")
```

Reply:
Might as well make it
```suggestion
        return print(io, MersenneTwister, "(", seed_str, ")")
```

Reply:
Is this only for allocation reasons?

Reply:
Yes, I would think so.